sudoku/src/main/java/de/sfuhrm/sudoku/output/AbstractTextFormatter.java

```diff
@@ -47,18 +51,32 @@ public final void setUnknownCellContentCharacter(
         this.unknownCellContentCharacter = newUnknownCellContentCharacter;
     }
 
-    /** Gets the line separator String to use.
-     * @return the line separator String.
+    /** Gets the row separator String to use.
+     * @return the row separator String.
+     */
+    public final String getRowSeparator() {
+        return rowSeparator;
+    }
+
+    /** Sets the row separator String to use.
+     * @param newRowSeparator the new String to use as row separator.
+     */
+    public final void setRowSeparator(final String newRowSeparator) {
+        this.rowSeparator = newRowSeparator;
+    }
+
+    /** Gets the column separator String to use.
+     * @return the column separator String.
      */
-    public final String getLineSeparator() {
-        return lineSeparator;
+    public final String getColumnSeparator() {
```

Comment:
It is better to keep the nomenclature in the project consistent (line -> row).
This is a breaking API change that needs a new major version number. I have ideas for the output package anyway going in this direction.